src/main/java/bizbook/logic/commands/AddNotesCommand.java

```diff
@@ -60,7 +59,7 @@ public CommandResult execute(Model model) throws CommandException {
         Person personToEdit = lastShownList.get(index.getZeroBased());
 
         // Update notes with new note
```

Comment:
It would be better to rename notesToEdit to noteToAdd  so as to not confuse with editNoteCommand. Could make note singular too since we are only modifying 1 note object at a time.